wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java

```diff
@@ -0,0 +1,240 @@
+
+        mListView = (WearableListView) findViewById(R.id.settings_list);
+        mAdapter = new WearableConfigAdapter(this, getConfigurations());
+        mListView.setAdapter(mAdapter);
+        mListView.addOnScrollListener(new WearableListView.OnScrollListener() {
+            @Override
+            public void onScroll(int i) {
+
+            }
+
+            @Override
+            public void onAbsoluteScrollChange(int scroll) {
+                float translation = Math.min(-scroll, 0);
+                headerText.setTranslationY(translation);
+            }
+
+            @Override
+            public void onScrollStateChanged(int i) {
+
+            }
+
+            @Override
+            public void onCentralPositionChanged(int i) {
+
+            }
+        });
+        mListView.setClickListener(new WearableListView.ClickListener() {
+            @Override
+            public void onClick(WearableListView.ViewHolder viewHolder) {
+                WearableListItemLayout layout = (WearableListItemLayout) viewHolder.itemView;
+
+                TextView nameTextView = (TextView) layout.findViewById(R.id.setting_text_view);
+                String action = nameTextView.getText().toString();
+                if (action.contains("Background") || action.contains("Hand") || action.contains("Marker")) {
+                    Intent intent = new Intent(WearableConfigurationActivity.this, ColorConfigActivity.class);
+                    intent.putExtra(ColorConfigActivity.CONFIG_HEADER, action);
+                    startActivityForResult(intent, REQUEST_COLOR);
+                } else if (CONFIG_DATE.equals(action)) {
+                    TextView settingTextView = (TextView) layout.findViewById(R.id.subsetting_text_view);
+                    CircledImageView circleImage = (CircledImageView) layout.findViewById(R.id.setting_circle);
+                    PutDataMapRequest putDataMapRequest = PutDataMapRequest.create(PATH_DATE);
+
+                    if (getString(R.string.label_setting_on).equals(settingTextView.getText().toString())) {
+                        settingTextView.setText(getString(R.string.label_setting_off));
+                        circleImage.setImageResource(R.drawable.ic_date_off);
+                        putDataMapRequest.getDataMap().putBoolean(action, false);
+                        mDisplayDate = false;
+                        updateConfigurations();
+                    } else {
+                        settingTextView.setText(getString(R.string.label_setting_on));
+                        circleImage.setImageResource(R.drawable.ic_date_on);
+                        putDataMapRequest.getDataMap().putBoolean(action, true);
+                        mDisplayDate = true;
+                        updateConfigurations();
+                    }
+
+                    PutDataRequest putDataRequest = putDataMapRequest.asPutDataRequest();
+                    Wearable.DataApi.putDataItem(mGoogleApiClient, putDataRequest);
```

Comment:
I think that they should be added when the mobile config screen is created. But if you don't have time to refactor, they can stay.